### ISSUE TITLE
Accuracy issues when reproducing on FERPlus dataset

### ISSUE THREAD

**Issue by chhchan** [zczcwh/POSTER#12]
Hello, When I tried to reproduce on the FERPlus dataset, I found the results were significantly different from your paper. Could you tell the hyper-parameter such as learning rate and batch size, to better reproduce experimental results? Thank you for your help.
